analysis/cnvs/prep_clinvar_cnvs.ipynb

```diff
@@ -269,32 +292,17 @@
     "### apply the error extraction function to our dataframe's column containing the errors\n",
     "df[\"error_string\"] = df[\"out.errors\"].fillna(\"\").apply(get_errors)\n",
     "df[\"error_string_reduce\"] = df[\"error_string\"].apply(reduce_errors)\n",
-    "df[\"error_string_reduce_2\"] = df[\"error_string_reduce\"].apply(reduce_errors_again)"
+    "#df[\"error_string_reduce_2\"] = df[\"error_string_reduce\"].apply(reduce_errors_again)"
```

Comment:
We shouldn't leave in commented out code. Recommend deleting the cells (here and other places)